tests/signals.cc

```diff
@@ -8,7 +8,8 @@ using namespace aspect;
 
 // create a function that is run upon loading the plugin
 // when declaring parameters, and that produces some output
-void declare_parameters(ParameterHandler &prm)
+void declare_parameters(const unsigned int dim,
+			ParameterHandler &prm)
```

Comment:
It's only in tests/ where the indentation script doesn't look. But I fixed it anyway.